models/Walk_2018/s15.0c_3D_fastrot_dir2_LS220_nux

```diff
@@ -0,0 +1,1013 @@
+0.10601 35.325 15.251 293.69
+0.10651 35.313 15.257 294
+0.10701 35.303 15.264 294.33
+0.10751 35.288 15.27 294.62
+0.10801 35.265 15.275 294.86
+0.10851 35.237 15.28 295.09
+0.10901 35.212 15.285 295.37
+0.10951 35.193 15.293 295.72
+0.11001 35.172 15.3 296.04
+0.11051 35.145 15.305 296.27
+0.11101 35.109 15.309 296.45
+0.11151 35.059 15.311 296.55
+0.11201 35.001 15.312 296.63
+0.11251 34.946 15.315 296.72
+0.11301 34.89 15.317 296.8
+0.11351 34.823 15.318 296.81
+0.11401 34.744 15.317 296.75
+0.11451 34.655 15.314 296.63
+0.11501 34.553 15.31 296.45
+0.11551 34.442 15.305 296.23
+0.11601 34.338 15.301 296.06
+0.11651 34.239 15.298 295.91
+0.11701 34.141 15.294 295.75
+0.11751 34.046 15.291 295.61
+0.11801 33.949 15.287 295.44
+0.11851 33.859 15.284 295.32
+0.11901 33.788 15.283 295.29
+0.11951 33.723 15.283 295.27
+0.12001 33.657 15.281 295.26
+0.12051 33.591 15.281 295.26
+0.12101 33.539 15.282 295.34
+0.12151 33.503 15.285 295.52
+0.12201 33.483 15.291 295.8
+0.12251 33.464 15.297 296.11
+0.12301 33.443 15.303 296.38
+0.12351 33.422 15.309 296.62
+0.12401 33.405 15.313 296.86
+0.12451 33.387 15.318 297.08
+0.12501 33.363 15.323 297.29
+0.12551 33.336 15.328 297.53
+0.12601 33.308 15.334 297.8
+0.12651 33.277 15.34 298.09
+0.12701 33.236 15.344 298.29
+0.12751 33.185 15.348 298.45
+0.12802 33.116 15.349 298.54
+0.12852 33.033 15.349 298.55
+0.12902 32.955 15.348 298.52
+0.12952 32.897 15.349 298.52
+0.13001 32.851 15.35 298.54
+0.13051 32.794 15.35 298.49
+0.13101 32.732 15.348 298.44
+0.13151 32.701 15.35 298.55
+0.13201 32.703 15.356 298.82
+0.13251 32.712 15.363 299.1
+0.13302 32.7 15.368 299.31
+0.13352 32.69 15.373 299.58
+0.13402 32.701 15.382 299.97
+0.13452 32.712 15.391 300.36
+0.13501 32.71 15.397 300.64
+0.13551 32.697 15.401 300.85
+0.13601 32.654 15.403 300.91
+0.13651 32.576 15.399 300.8
+0.13701 32.499 15.397 300.7
+0.13751 32.437 15.397 300.7
+0.13801 32.35 15.395 300.57
+0.13851 32.243 15.388 300.23
+0.13901 32.129 15.378 299.78
+0.13951 32.008 15.367 299.25
+0.14 31.891 15.357 298.75
+0.14 31.891 15.357 298.75
+0.1405 31.779 15.347 298.27
+0.141 31.675 15.339 297.85
+0.1415 31.568 15.331 297.47
+0.142 31.465 15.324 297.18
+0.1425 31.382 15.321 297.05
+0.143 31.337 15.322 297.1
+0.14351 31.31 15.324 297.17
+0.14401 31.265 15.322 297.13
+0.14451 31.227 15.322 297.17
+0.145 31.219 15.328 297.5
+0.1455 31.268 15.343 298.18
+0.146 31.385 15.365 299.16
+0.1465 31.52 15.386 300.12
+0.147 31.612 15.4 300.77
+0.1475 31.634 15.406 301.08
+0.148 31.628 15.411 301.31
+0.1485 31.632 15.42 301.73
+0.149 31.637 15.429 302.13
+0.1495 31.632 15.435 302.43
+0.15 31.589 15.439 302.57
+0.1505 31.554 15.443 302.68
+0.151 31.511 15.445 302.72
+0.1515 31.471 15.445 302.7
+0.152 31.389 15.441 302.45
+0.1525 31.267 15.432 301.97
+0.153 31.151 15.423 301.51
+0.1535 31.065 15.418 301.19
+0.154 30.974 15.411 300.82
+0.1545 30.828 15.398 300.2
+0.155 30.68 15.384 299.61
+0.15532 30.612 15.379 299.35
+0.15532 30.611 15.379 299.35
```

Comment:
… or here. And the same duplicates also appear in other files. Could you please run [this code snippet](https://github.com/SNEWS2/snewpy/pull/161#issuecomment-1005621087) to find and remove these duplicates?

Reply:
oh okey, I just checked at the end of the file were previously there were the duplicates, I did not realise that was also occurring in the middle, I will run it assap